std/http/file_server_test.ts

```diff
@@ -262,15 +270,16 @@ Deno.test("partial TLS arguments fail", async function (): Promise<void> {
       "run",
       "--allow-read",
       "--allow-net",
-      "http/file_server.ts",
+      "file_server.ts",
       ".",
       "--host",
       "localhost",
       "--cert",
-      "./http/testdata/tls/localhost.crt",
+      "./testdata/g/tls/localhost.crt",
```

Comment:
Is `g/` intentional or a typo?

Reply:
Thats a substitution gone bad.

```suggestion
      "./testdata/tls/localhost.crt",
```